docs/conf.py

```diff
@@ -63,3 +70,54 @@
     "examples_dirs": "../examples",  # path to your example scripts
     "gallery_dirs": "auto_examples",  # path to where to save gallery generated output
 }
+
+
+def linkcode_resolve(domain, info):
```

Comment:
maybe add reference to original implementation here?

Reply:
yes, thanks, added it.